### PR TITLE
Update create-a-phone-system-auto-attendant.md

### PR DESCRIPTION
**Pull request by CLYVR**
Clarifying external phone number formats

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-a-phone-system-auto-attendant.md [hunk 1]
```diff
@@ -194,7 +194,10 @@ When you have finished adding service accounts, click **Submit**. This completes
 When transferring calls to an external phone number, the resource account associated with the auto attendant or call queue must have a phone number and a Microsoft 365 Phone System - Virtual User license. Additionally:
 
 - For a resource account with a Calling Plan number, assign a [Calling Plan](calling-plans-for-office-365.md) license.
+  - The external transfer phone number must be entered in E.164 format (+CC+phone_numer)
```
Comment (JohanFreelancer9) [inline Teams/create-a-phone-system-auto-attendant.md:197]:
```suggestion
  - The external transfer phone number must be entered in E.164 format (+CC+phone_numer).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/create-a-phone-system-auto-attendant.md (hunk 1 above)
Comment (CLYVR) [reply Teams/create-a-phone-system-auto-attendant.md:197]: Actually, I think we both made the same typo - that should be num**b**er and not numer:  (+CC+phone_number)

Comment (JohanFreelancer9) [reply Teams/create-a-phone-system-auto-attendant.md:197]: Well spotted - thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
@CLYVR what's SBC?

**Comment by CLYVR** [comment]
@MikePlumleyMSFT SBC = [Session Border Controller](https://docs.microsoft.com/en-us/microsoftteams/direct-routing-connect-the-sbc)

**Comment by MikePlumleyMSFT** [comment]
Thanks @CLYVR . Added that link as well.

**Pull request merged**